src/server/services/procedures/wildcam/readme.md

```diff
@@ -0,0 +1,7 @@
+## Image Backups
+
+As a backup plan in case the data source were to ever be taken offline, the [`downloader.py`](downloader.py) script
+was written to quickly download all the images (with metadata already being saved in the CSV file itself).
+This was run once and stored in google drive long-term, but the script remains in case the image source is ever updated.
+At the time of writing, the total image dataset is around 3.3GB.
+Obviously, don't commit the downloaded image files to the repository...
```

Comment:
It might not be bad to download them all to a (hard-coded) save directory that is added to a gitignore file.